core/storage/exploration/gae_models.py

```diff
@@ -323,19 +323,14 @@ def compute_models_to_commit(
             additional_models
         )
 
-        # The cast is needed because the additional_models is list of BaseModels
-        # and we want to hint the mypy that this is ExplorationRightsModel.
-        exploration_rights_model = cast(
-            ExplorationRightsModel, additional_models['rights_model']
-        )
         exploration_commit_log = ExplorationCommitLogEntryModel.create(
             self.id, self.version,
             committer_id,
             commit_type,
             commit_message,
             commit_cmds,
-            exploration_rights_model.status,
-            exploration_rights_model.community_owned
```

Comment:
Why do you change this?

Reply:
Trying to pass hardcode value, as while running the beam job the `exploration_rights_model` is coming none for me.

Reply:
Beam job error
```
"rights_model [while running 'Generate exploration models to put']"
```

From beam job
```py
models_to_put = updated_exploration_model.compute_models_to_commit(
    feconf.MIGRATION_BOT_USERNAME,
    feconf.COMMIT_TYPE_EDIT,
    commit_message,
    change_dicts,
    additional_models={}
```
Should we somehow access the ``ExplorationRightsModel` and pass it to the addition_models above? 
Example
```py
exp_rights = (
    exp_models.ExplorationRightsModel.get_multi(exp_model.id))
```

core/storage/exploration/gae_models
```py
# The cast is needed because the additional_models is list of BaseModels
# and we want to hint the mypy that this is ExplorationRightsModel.
exploration_rights_model = cast(
    ExplorationRightsModel, additional_models['rights_model']
)
exploration_commit_log = ExplorationCommitLogEntryModel.create(
    self.id, self.version,
    committer_id,
    commit_type,
    commit_message,
    commit_cmds,
    exploration_rights_model.status,
    exploration_rights_model.community_owned
)
```

Reply:
When you call `compute_models_to_commit`, you need to pass the appropriate `ExplorationRightsModel` in the `additional_models`. There can't be any modifications to this code.